sys/socket_unix.go

```diff
@@ -28,6 +28,11 @@ func GetLocalListener(path string, uid, gid int) (net.Listener, error) {
 		return l, err
 	}
 
+	if err := os.Chmod(path, 0770); err != nil {
```

Comment:
Previously it was `0755`, but I don't think the execute permission has any purpose here so we can drop it.